index.js

```diff
@@ -90,5 +90,11 @@ AnimationLoop.prototype.stop = function () {
 	}
 };
 
-// Expose
-module.exports = AnimationLoop;
\ No newline at end of file
+var instance;
+
+module.exports = (function(){
```

Comment:
Shouldn't it be `module.exports = function()`? Otherwise, you may as well convert this whole PR to `module.exports = new AnimationLoop();`